openslides_backend/action/group/set_permission.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any, Dict
+
+from ...models.models import Group
+from ...shared.patterns import FullQualifiedId
+from ..default_schema import DefaultSchema
+from ..generics import UpdateAction
+from ..register import register_action
+
+
+@register_action("group.set_permission")
+class GroupSetPermissionAction(UpdateAction):
+    """
+    Action to set permission of a group.
+    """
+
+    model = Group()
+    schema = DefaultSchema(Group()).get_update_schema(
+        additional_required_fields={
+            "set": {"type": "boolean"},
+            "permission": {"type": "string"},
+        }
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        set_ = instance.pop("set")
+        permission = instance.pop("permission")
+        group = self.datastore.get(
+            FullQualifiedId(self.model.collection, instance["id"]), ["permissions"]
+        )
+        instance["permissions"] = group["permissions"]
```

Comment:
I would suggest to only set the `permissions` field if something was changed, so we don't create unnecessary write requests.

Reply:
Done, if the permissions are not changed, the instance is not updated.